### ISSUE TITLE
Attentional Graph Neural Network for Parking-Slot Detection

### ISSUE THREAD

**Issue by Ordinary-child** [Jiaolong/gcn-parking-slot#1]
Hello, does the given code have to be trained and tested on GPU

**Comment by Jiaolong** [comment]
Yes, it requires GPU to train. For testing and inference, you can try CPU.

**Comment by Ordinary-child** [comment]
你可以发一份论文文档给我吗 ？我在网上下载不了

**Comment by brankyy** [comment]
> Yes, it requires GPU to train. For testing and inference, you can try CPU.

您好，如果想进行testing和inference，怎么设置部署在CPU呢？设置launcher并不能成功运行
